content/en/docs/concepts/architecture/nodes.md

```diff
@@ -21,7 +21,7 @@ architecture design doc for more details.
 
 {{% capture body %}}
 
-## Node Status
+### Node Status
```

Comment:
It should be `##` here because the below `### Addresses` (and others) are parts of it.

Reply:
My thought was to keep in line with the new style guide i.e. prometheus would be acceptable, but the GKE specific items would not. Possibly keep the linuxacademy CKA classes, but remove the Oracle related items, etc. 

re: #15576 

/cc @zacharysarah

Reply:
Not sure this commit should be part of this PR

Reply:
Yes, whoops, sorry, I'll undo the change.